### PR TITLE
Completed css-canvas-medical exercise. - Davin Phuong

### PR DESCRIPTION
**Pull request by dn-phuong**
css-canvas-medical

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Davin! Your "Get Started" button seems to be vertically misaligned with the "Ready to learn more?" text section, and the final row with the two anchor links is too short as compared to the example, please revise and submit a new screenshot with your revision.

**Comment by dn-phuong** [comment]
css-canvas-medical 2

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by dn-phuong** [comment]
Screen Shot 2022-02-09 at 8 24 47 PM

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+  width: 38%;
+  padding-top: 15%;
+  text-align: left;
+  font-size: 17px;
+  line-height: 2;
+  color: white;
+  margin: 5% 8%;
+  display: inline-block;
+}
+
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:168]: Try doing this without positioning the text, heading, or button. Instead, try putting them inside of the overlay and using padding to push them inward from the edges of the overlay.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
No rush on this, we just want to avoid overusing positioning.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**